.changeset/new-mangos-tap.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/cli': patch
+---
+
+Tweak error message in lockfile parsing to include more information.
```

Comment:
"lockfile" needs to be added to https://github.com/backstage/backstage/blob/master/.github/styles/vocab.txt for the failing check. 😁